python/hsfs/constructor/query.py

```diff
@@ -146,6 +147,12 @@ def read(
         # Returns
             `DataFrame`: DataFrame depending on the chosen type.
         """
+        if not isinstance(online, bool):
+            warnings.warn(
```

Comment:
why only  a warning and not throw an exception here?

Reply:
To avoid breaking change